spec.emu

```diff
@@ -236,13 +236,16 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _style_ be ? GetOption(_options_, _unit_, *"string"*, _stylesList_, *undefined*).
         1. Let _displayDefault_ be *"always"*.
         1. If _style_ is *undefined*, then
-          1. Set _displayDefault_ to *"auto"*.
```

Comment:
Suggestion: A smaller change would be to replace this line with

```
1. If _baseStyle_ is not *"digital"* or _unit_ is not one of *"hours"*, *"minutes"*, or *"seconds"*, then
    1. Set _defaultDisplay_ to *"auto"*.
```

and no other changes are necessary.

Reply:
I think the "A is **not** B or C is **not** D"  kind of statment is very confusing to read and mentally easier to get misread.

Reply:
Another way to write it is

```
1. If _baseStyle_ is *"digital"* and _unit_ is one of *"hours"*, *"minutes"*, or *"seconds"*, then
    1. Set _defaultDisplay_ to *"always"*.
1. Else,
    1. Set _defaultDisplay_ to *"auto"*.
```

Reply:
btw, I remember "!(A && B) == !A || !B" as one of the very first things I learned in my freshman computer science class in college.

Reply:
The problem is " not *"digital"* OR _unit_ is not ONE OF *"hours"*, *"minutes"*, OR *"seconds"*"  have two differet kinds of OR in that statment, a logical OR and a set OR (the OR in "ONE Of A, B, OR C") which is confusing to read.
I just think one "if statment" contains both a "logical and / or" and a "set base or" is very confusing. It is not difficult to read if statment with not and or, but mixing logical expression with set expression is just not very straightforward